### ISSUE TITLE
Event-values does not work

### ISSUE THREAD

**Issue by Vulcano771** [SkriptLang/skript-reflect#103]
I updated to the latest version of sk-reflect and now I get:
```
[11:43:36 INFO]: [Skript] Line 60: (test.sk)
[11:43:36 INFO]:     There's no player in an on custom event event
```

It worked before.

```
event "test":
	pattern: test
	event-values: player, world
	check:
		continue
```

**Comment by sovdeeth** [comment]
Should already be fixed by #92, just not in release 2.4

**Comment by sovdeeth** [comment]
Confirmed fixed by 92
